src/Microsoft.AspNetCore.Antiforgery/ServiceCollectionExtensions.cs

```diff
@@ -10,9 +10,16 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
+    /// Extension methods for setting up antiforgery services in an <see cref="IServiceCollection" />.
+    /// </summary>
     public static class ServiceCollectionExtensions
     {
-        public static IServiceCollection AddAntiforgery(this IServiceCollection services)
+        /// <summary>
+        /// Adds antiforgery services to the specified <see cref="IServiceCollection" />.
```

Comment:
Consider it forgotten. I prefer `<paramref/>` because that's more consistent with `<summary>`s throughout MVC. I also think making the comments word-for-word identical across our repos should be a non-goal.